src/templates/app/user/security/self_exclusion.jsx

```diff
@@ -78,6 +78,16 @@ const SelfExclusion = () => (
                 </div>
 
                 <SubmitButton text={it.L('Update Settings')} type='submit' />
+
+                <div id='ukgc_requirement_notice' data-show='iom, malta' className='gr-row invisible'>
+                    <div className='gr-8 gr-push-4 gr-12-m gr-push-0-m'>
+                        <div className='notice-msg gr-child gr-parent'>
+                            <p>{it.L('Self-exclusion on this website only applies to your [_1] account and does not include other companies or websites.', it.website_name)}</p>
+                            <p>{it.L('Since you feel the need to self-exclude, we strongly encourage you to exclude yourself from similar services that you may have signed up for.')}</p>
+                            <p>{it.L('Thank you for trading with [_1].', it.website_name)}</p>
```

Comment:
remove the "Thank you for trading with Binary.com" line. The client is clearly unhappy with the service as he is self-excluding, so it's a bit tone-deaf to be saying thank you for trading..

Reply:
sure, will do